src/features/Treege/components/Forms/ReceiveValueFromAncestor.tsx

```
@@ -0,0 +1,62 @@
+
+const ReceiveValueFromAncestor = ({ onChange, id, value, ancestors }: ReceiveValueFromParentProps) => {
+  const { t } = useTranslation(["form"]);
+  const { tree, currentHierarchyPointNode } = useTreegeContext();
+  const { uuid } = currentHierarchyPointNode?.data || {};
+
+  if (!tree || !uuid) {
+    return null;
+  }
+
+  const handleChange = (event: SelectChangeEvent<string | undefined>) => {
+ const { value: newValue } = event.target;
```

ESLint

[error] Insert `···`

(prettier/prettier)